examples/Results/main.roc

```diff
@@ -0,0 +1,106 @@
+app [main] {
+    pf: platform "https://github.com/roc-lang/basic-cli/releases/download/0.12.0/Lb8EgiejTUzbggO2HVVuPJFkwvvsfW6LojkLR20kTVE.tar.br",
+}
+
+import pf.Stdout
+import pf.Task exposing [Task]
+
+# This function parses a string formatted like "{FirstName} {LastName} was born in {Year}"
+# This is the most verbose version. It's a bit too nested, we will do better below.
+parsePersonVerbose = \line ->
```

Comment:
Can we add a version of this use `Result.try` and `Result.mapErr`, but no backpassing? I think that will be useful to new users.

Reply:
Good suggestion, thanks. I've added `parseWithTryV2`.